Source/WebCore/layout/formattingContexts/inline/text/TextBreakingPositionCache.cpp

```diff
@@ -97,7 +97,7 @@ void TextBreakingPositionCache::clear()
 
 void add(Hasher& hasher, const TextBreakingPositionContext& context)
 {
-    add(hasher, context.whitespace, context.overflowWrap, context.lineBreak, context.wordBreak, context.nbspMode, context.locale);
+    add(hasher, context.whiteSpaceCollapseBehavior, context.overflowWrap, context.lineBreak, context.wordBreak, context.nbspMode, context.locale);
```

Comment:
nit: do you mind lowercasing whiteSpace to make it blend in with the rest of IFC codebase?